crates/telio-wg/src/wg.rs

```diff
@@ -1377,6 +1384,13 @@ pub mod tests {
         // Connect
         peer.time_since_last_handshake = Some(Duration::from_secs(15));
         ifa.peers.insert(pkc, peer.clone());
+
+        let _ = task_exec!(&wg.task, async move |s| {
```

Comment:
Why are we goint into internals of implemenation to update stats, this will make this test very fragile.
Can you do this through update, and test a more realistic case

Reply:
Fixed